### ISSUE TITLE
Any key on value generates error Assert failure"" failed in wxGetKeyStateGTK(): Unsupported key 391

### ISSUE THREAD

**Issue by camposFabio** [moneymanagerex/moneymanagerex#6387]
MMEX version:
 - [ ] 1.6.4
 - [x] 1.7.0
 - [ ] 1.7.1
 - [ ] 1.7.2
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [] Windows
 - [ ] Mac OSX
 - [X] Linux 

Location information:
GDM_LANG=pt_BR.UTF-8
LANG=pt_BR.UTF-8

-- 
-- MMEX configuration summary
-- ==========================
-- Version        : 1.7.0
-- Commit         : b38049357 (2023-12-26)
-- Host system    : Linux x86_64
-- Target system  : Linux x86_64
-- Build types    : 
-- Generator      : Unix Makefiles
-- Install prefix : /usr
-- DB encryption  : ON
-- 
-- Versions
-- --========--
-- Linux 6.6.8-arch1-1
-- Arch.rolling
-- Arch.Linux
-- CMake 3.28.1
-- GNU Make 4.4.1
-- ccache
-- GNU 13.2.1
-- wxWidgets 3.2.4
-- wxSQLite3 4.9.1
-- RapidJSON 1.1.0
-- Lua 5.4.6
-- curl 8.5.0
-- gettext 0.22.4
-- ------------------ 8< -----------------
-- Configuring done (1.0s)
-- Generating done (0.0s)

# Description of the bug

When entering an new transaction any number I enter on Value field generates the following error:

/usr/src/debug/wxwidgets/wxWidgets-3.2.4/src/unix/utilsx11.cpp(2645): assert ""Assert failure"" failed in wxGetKeyStateGTK(): Unsupported key 391, the only supported ones are: Ctrl, Alt, Shift, Caps Lock, Num Lock and Scroll Lock for GTK 3.18+

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

Create an new transaction and try to enter an new value

# Additional information

If you have a backtrace for a crash or a warning, paste it here.

![Captura de tela de 2023-12-27 11-32-07](https://github.com/moneymanagerex/moneymanagerex/assets/40764894/2dd8c6f3-26a7-45ff-9811-3aaf044e4065)

**Comment by whalley** [comment]
https://github.com/moneymanagerex/moneymanagerex/issues/4895 related?

**Comment by camposFabio** [comment]
I don´t know

**Comment by camposFabio** [comment]
Solved compiling WxWidgets without debug information:

`../configure BUILD=release --with-gtk=3  --enable-stl --disable-debug`

Related to the issue #6458

**Comment by nasko** [comment]
Hey guys!

I'm experiencing this very issue with moneymanagerex 1.8.0 under Arch Linux , even after compiling wxwidgets using the `--disable-debug` flag, as suggested by @camposFabio.

I performed these steps without seeing any errors:

- I verified that I had these two packages installed:
  - extra/wxwidgets-common 3.2.5-1
  - extra/wxwidgets-gtk3 3.2.5-1
- Navigated to the [wxwidgets Gitlab repo](https://gitlab.archlinux.org/archlinux/packaging/packages/wxwidgets) and copied the git clone URL: https://gitlab.archlinux.org/archlinux/packaging/packages/wxwidgets.git
- Cloned the repo
- Modified PKGBUILD to include the `--disable-debug` flag. This is the final configure command, before compiling: `./configure --prefix=/usr --disable-tests --disable-debug`
- Built the packages with `makepkg -s`. The output were these four files:
  - wxwidgets-common-3.2.5-1-x86_64.pkg.tar.zst
  - wxwidgets-debug-3.2.5-1-x86_64.pkg.tar.zst
  - wxwidgets-gtk3-3.2.5-1-x86_64.pkg.tar.zst
  - wxwidgets-qt5-3.2.5-1-x86_64.pkg.tar.zst
- Installed the three, skipping wxwidgets-debug, using `pacman -U`
- Restarted my laptop
- Fired MoneyManagerEx
- Brought up the new transaction window
- Started typing in the Amount field

The error was immediately shown. [Relevant backtrace](https://gist.github.com/nasko/a0c1b8977b6c80e61113ec2819b02f0e)

I'd be grateful for any hints as to how I can resolve this. There is an option to continue without seeing this error for the duration of the active session, but another issue (that I'm not sure if it's related to the same bug) is that typing in the Category- or Payee fields doesn't automatically suggest and narrow-down the available options for easier selecting of the desired entry.

Thanks!
